code/modules/bodytype/bodytype_offsets.dm

```diff
@@ -18,15 +18,19 @@ The slots that you can use are found in items_clothing.dm and are the inventory
 */
```

Comment:
might be nice to note you can use `BP_R_HAND` etc for held items as well? i think?
also maybe would be good to explain why we need `(foo)` as the key. todo `FOOTGUNS.MD` i guess